components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -1081,14 +1082,45 @@ public int compare(Tag o1, Tag o2) {
 		                                APIConstants.TAGS_INFO_ROOT_LOCATION +
 		                                        "/%s/description.txt";
 		String thumbnailPathPattern = APIConstants.TAGS_INFO_ROOT_LOCATION + "/%s/thumbnail.png";
+
+		//if the tenantDomain is not specified super tenant domain is used
+		if(tenantDomain == null || tenantDomain.trim() == "" ){
```

Comment:
Fixed in #1494(https://github.com/wso2/carbon-apimgt/commit/b672e7ec2a5973ae34af2999490837d9333cafc4)